instructor_notes/11_git-init-server.md

```diff
@@ -0,0 +1,116 @@
+# Instructor Notes: Initializing a git repository on RStudio Server
+
+This document provides instructions for creating a git repository in an existing directory on the Data Lab's RStudio Server, as well as setting up associated git credentials.
+
+## Learning goals
+
+At the end of this activity, workshop attendees will have:
+
+* Learned about git Personal Access Tokens and credentials
+* Learned basic git configuration using R tools
+* Used basic git CLI commands
+
+## Part 1: Set up git credentials
+
+### Set up the `~/.gitconfig` file
+
+This can either be done directly in the editor by opening `~/.gitconfig`, or with `usethis` which may be safer for ensuring correct spacing:
+
+In the end, the `~/.gitconfig` file should look like this (where the `init` block is optional but recommended):
```

Comment:
Do we want to suggest using `git config` to do this? Somewhat more typing, but 
```
git config --global user.email "your_email@example.com"
git config --global user.name "name"
git config --global credential.helper "cache --timeout=43200"
```

The main reason to do this is that it will always write to the correct file, which may not always be `.gitconfig` (though it should be at ~/.gitconfig in this case)

This still does not validate typos though